public/app/features/alerting/unified/components/rules/AlertInstanceStateFilter.tsx

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
+import { capitalize } from 'lodash';
+import React, { useMemo } from 'react';
 
-import { RadioButtonGroup, Label } from '@grafana/ui';
-import { GrafanaAlertState } from 'app/types/unified-alerting-dto';
+import { Label, RadioButtonGroup } from '@grafana/ui';
+import { GrafanaAlertState, PromAlertingRuleState } from 'app/types/unified-alerting-dto';
+
+export type InstanceStateFilter = GrafanaAlertState | PromAlertingRuleState.Pending | PromAlertingRuleState.Firing;
 
 interface Props {
   className?: string;
-  stateFilter?: GrafanaAlertState;
-  onStateFilterChange: (value: GrafanaAlertState | undefined) => void;
+  filterType: 'grafana' | 'prometheus';
+  stateFilter?: InstanceStateFilter;
+  onStateFilterChange: (value: InstanceStateFilter | undefined) => void;
```

Comment:
nit: This is existing code but I think in most places we just mark the input param as optional.